docs/src/vim.md

```diff
@@ -48,12 +48,14 @@ g <   The same, but backwards
 g a   Add a visual selection for every copy of the current word
 
 # Pane management
+g /        Open a project-wide search
 g <space>  Open the current search excerpt
 <ctrl-w> <space>  Open the current search excerpt in a split
 <ctrl-w> g d      Go to definition in a split
 <ctrl-w> g D      Go to type definition in a split
 
 # Insert mode
+i a / a a      Select the function argument the cursor is in
```

Comment:
？This seems to be the wrong commit

Reply:
definitely. I was reviewing the file and added it (I didn't document gu/gU/g~ because they are in vim).